svcb.go

```diff
@@ -511,8 +511,13 @@ func (s *SVCBIPv4Hint) parse(b string) error {
 }
 
 func (s *SVCBIPv4Hint) copy() SVCBKeyValue {
+	hint := []net.IP(nil)
```

Comment:
I’d prefer:

```go
hint := make([]net.IP, len(s.Hint))
for i, ip := range s.Hint {
hint[i] = copyIP(ip)
}

return &SVCBIPv4Hint{hint}
```

Reply:
I was thinking about it, but we'll end up with an empty array in the `s.Hint = nil` case which is a little bit different from the original. Is it okay?

Reply:
Yeah that should be fine. It’s rare for such a distinction to matter. It’s much faster to preallocate and avoid append where that’s easy to do.